TFG-Labs/custom-shipping-step-by-items
# Feature/tlva 58/test combo

### What problem is this solving?

Functionality developed to satisfy customer requirements for mixed products to display an alert for the custommer.

#### How it works

If there are products with furniture, simcard or television category in the shopping cart, the customer will be shown an alerts and different forms.

#### How to test it?

[Workspace](https://tlva58--thefoschini.myvtex.com/)

You must be logged in with a customer already registered on the platform.

In the checkout add the product with SKU **1, 59418103 and 59418159**.

When you reach the Shipping step and you have entered the shipping address, the alert and forms will appear as you can see in the following video.

#### How does this PR make you feel? 🔗

![](https://giphy.com/clips/justin-reaction-meme-realization-poOsWEQNzdORIiJ8NH)
